AgentDiscoveries-Frontend/app/end_to_end_tests/login_test.js

```diff
@@ -0,0 +1,113 @@
+import chai from 'chai'
+import React from 'react'
+import * as webdriver from 'selenium-webdriver'
+import * as chrome from 'selenium-webdriver/chrome'
+//The next import is required to ensure chromedriver is on the system path
+import * as chromeDriver from 'chromedriver';
```

Comment:
Are we running these tests on Jenkins?

I'll be curious to see if ChromeDriver works - I have had many problems in the past with this, but maybe we're using a modern enough server for it to be fine. :)

Reply:
The java dummy test which just checked the page title ran on Jenkins using HtmlUnitDriver
I read that ChromeDriver was usually ok as long as you ran it headless, but I haven't verified this.

Reply:
Alright cool, thanks. I know it has issues on old versions of linux but hopefully it'll Just Work (TM)